src/ds/bits.h

```diff
@@ -225,7 +235,7 @@ namespace snmalloc
 
       for (int i = BITS - 1; i >= 0; i--)
```

Comment:
```C
for (int i = BITS - 1; i >= 0; i--)
```
should be 
```C
for (size_t i = BITS - 1; i >= 0; i--)
```

Reply:
I don't know what this is trying to do, but that change make this an infinite loop.  Specifically, an infinite loop at compile time in a constexpr that's used as a template parameter...

Reply:
Sorry, there was a int -> size_t conversion complaint from this line.